server/datastore/mysql/software_installers_test.go

```diff
@@ -735,6 +735,41 @@ func testBatchSetSoftwareInstallers(t *testing.T, ds *Datastore) {
 		{Name: ins1, Source: "apps", Browser: ""},
 	})
 
+	// mark ins0 as install_during_setup
+	ExecAdhocSQL(t, ds, func(q sqlx.ExtContext) error {
+		_, err := q.ExecContext(ctx, ` UPDATE software_installers SET install_during_setup = 1 WHERE title_id = ?`, *softwareInstallers[0].TitleID)
+		return err
+	})
```

Comment:
Note that for now I only added database tests as the endpoints and logic to mark a software as "install during setup" is not ready yet. Integration tests should be added for the complete flow (using the endpoints as much as possible) when all the parts are ready. I think `ExecAdhocSQL` is not too bad for DB tests (doesn't mock/bypass a whole lot of logic), but best avoided if possible in integration tests.